### PR TITLE
Updated CHANGELOG.md.

### PR DESCRIPTION
**Pull request by russjones**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,12 +1,57 @@
...
+Teleport 7.0 is a major release of Teleport that contains new features, improvements, and bug fixes.
 
-Teleport 7.0 is a major release with new features, functionality, and bug fixes.
+## New Features
+
+### MongoDB 
+
+Added support for [MongoDB](https://www.mongodb.com) to Teleport Database Access. [#6600](https://github.com/gravitational/teleport/issues/6600).
+
+View the [Database Access with MongoDB](https://goteleport.com/docs/ver/7.0/database-access/guides/mongodb-self-hosted/ for more details.
```
Comment (r0mant) [inline CHANGELOG.md:11]:
```suggestion
View the [Database Access with MongoDB](https://goteleport.com/docs/ver/7.0/database-access/guides/mongodb-self-hosted/) for more details.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,12 +1,57 @@
...
+
+### AWS Console
+
+Added support for [AWS Console](https://aws.amazon.com/console) to Teleport Application Access. [#7590](https://github.com/gravitational/teleport/pull/7590)
+
+Teleport Application Access can now generate and automatically sign into the AWS Management Console using [Identity federation](https://aws.amazon.com/identity/federation).
```
Comment (r0mant) [inline CHANGELOG.md:17]:
```suggestion
Teleport Application Access can now automatically sign users into the AWS Management Console using [Identity federation](https://aws.amazon.com/identity/federation). View AWS Management Console [guide](https://goteleport.com/docs/ver/7.0/application-access/guides/aws-console/) for more details.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,12 +1,57 @@
...
 
-Teleport 7.0 is a major release with new features, functionality, and bug fixes.
+## New Features
+
+### MongoDB 
+
+Added support for [MongoDB](https://www.mongodb.com) to Teleport Database Access. [#6600](https://github.com/gravitational/teleport/issues/6600).
+
+View the [Database Access with MongoDB](https://goteleport.com/docs/ver/7.0/database-access/guides/mongodb-self-hosted/ for more details.
+
```
Comment (r0mant) [inline CHANGELOG.md:12]:
```suggestion

### Cloud SQL MySQL

Added support for [GCP Cloud SQL MySQL](https://cloud.google.com/sql/docs/mysql) to Teleport Database Access. [#7302](https://github.com/gravitational/teleport/pull/7302)

View the Cloud SQL MySQL [guide](https://goteleport.com/docs/ver/7.0/database-access/guides/mysql-cloudsql/) for more details.

```

**Review by r0mant — APPROVED** [review]
Couple suggestions, otherwise lgtm.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,12 +1,57 @@
...
+## Improvements
+
+* Added the ability to terminate Database Access certificates when the certificate expires. [#5476](https://github.com/gravitational/teleport/issues/5476)
+* Added additional FedRAMP compliance controls, such as custom disconnect and MOTD messages. [#6091](https://github.com/gravitational/teleport/issues/6091) [#7396](https://github.com/gravitational/teleport/pull/7396)
+* Added the ability to export Audit Log and session recordings using the Teleport API. [#6731](https://github.com/gravitational/teleport/pull/6731) [#7360](https://github.com/gravitational/teleport/pull/7360)
+* Added the ability to partially configure a cluster. [#5857](https://github.com/gravitational/teleport/issues/5857) [RFD #28](https://github.com/gravitational/teleport/blob/master/rfd/0028-cluster-config-resources.md)
+* Added the ability to disable port forwarding on a per-host basis. [#6989](https://github.com/gravitational/teleport/pull/6989)
+* Added ability to configure `tsh` home directory. [#7035](https://github.com/gravitational/teleport/pull/7035/files)
```
Comment (benarent) [inline CHANGELOG.md:34]:
```suggestion
* Added ability to generate OpenSSH client configuration snippets using `tsh config`. [#7437](https://github.com/gravitational/teleport/pull/7437)
```

File: CHANGELOG.md (hunk 2 above)
Comment (benarent) [reply CHANGELOG.md:17]: @r0mant Should we make this [guide](https://goteleport.com/docs/application-access/guides/aws-console/) as we'll push docs at the same time?

**Review by klizhentas — APPROVED** [review]

**Pull request merged**